package.json

```diff
@@ -58,7 +58,7 @@
     "test:js": "mocha --require src/lib/test-setup.js --recursive \"src/**/*-test.js\"",
     "deploy": "gh-pages -x -d build/web -r \"https://$GH_TOKEN@github.com/mozilla/FirefoxColor.git\"",
     "release:base": "npm-run-all clean build:web sign && mv addon.xpi build/web && npm run deploy",
-    "release:dev": "cross-env ADDON_URL='https://addons-dev.allizom.org/firefox/addon/firefox-color/' SITE_URL='https://mozilla.github.io/FirefoxColor/' SITE_ID=github npm run release:base"
```

Comment:
This URL is closer to production than a direct link to the xpi, but neither of the options work because the xpi needs to be signed, and CircleCI does currently not have valid credentials to generate a signed xpi file. I thought of removing the signing step, in favor of linking to unsigned files + clear instructions on how to load it (#845). But that requires #845 to be resolved first.